webapp/src/components/App/Singlestudy/explore/Results/ResultDetails/index.tsx

```diff
@@ -193,99 +223,123 @@ function ResultDetails() {
           />
         }
         right={
-          <Box
-            sx={{
-              display: "flex",
-              flexDirection: "column",
-              height: 1,
-              width: 1,
-              gap: 1,
-            }}
-          >
+          isSynthesis ? (
```

Comment:
`right={
  <Box sx={...}>
    isSynthesis ? <Paper>.... : (....)
  </Box>
}`

The parent is the same

Reply:
Understood. I did it that way because the display of synthesis needed a different parent styles. I will try to have both in same parent but some styles are different